node/src/GlideClusterClient.ts

```diff
@@ -162,6 +163,36 @@ export type GlideClusterClientConfiguration = BaseClientConfiguration & {
  */
 export type ClusterResponse<T> = T | Record<string, T>;
 
+/**
+ * @internal
+ * Type which returns GLIDE core for commands routed to multiple nodes.
+ * Should be converted to {@link ClusterResponse}.
+ */
+type ClusterGlideRecord<T> = GlideRecord<T> | T;
+
+/**
+ * @internal
+ * Convert {@link ClusterGlideRecord} to {@link ClusterResponse}.
+ *
+ * @param res - Value received from Glide core.
+ * @param isRoutedToSingleNodeByDefault - Default routing policy.
+ * @param route - The route.
+ * @returns Converted value.
```

Comment:
we can be more detailed here:  it'll return the value if routed to a single node, or a Record of nodes to values if routed to multiple nodes.